tools/plugin-base/src/main/java/io/spine/tools/gradle/project/DependencyContainer.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2019, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.tools.gradle.project;
+
+import io.spine.tools.gradle.Artifact;
+import io.spine.tools.gradle.ConfigurationName;
+import io.spine.tools.gradle.Dependency;
+
+import static io.spine.tools.gradle.ConfigurationName.COMPILE;
+import static io.spine.tools.gradle.ConfigurationName.IMPLEMENTATION;
+
+/**
+ * A container of dependencies of a certain project.
+ *
+ * <p>Typically, represented by a {@link org.gradle.api.artifacts.dsl.DependencyHandler} of
+ * the project.
+ */
+public interface DependencyContainer {
+
+    /**
+     * Adds a new dependency within a given configuration.
+     *
+     * @param configurationName
+     *         the name of the Gradle configuration
+     * @param notation
+     *         the dependency string, e.g. {@code "io.spine:spine-base:1.0.0"}
+     */
+    void depend(ConfigurationName configurationName, String notation);
```

Comment:
If this is a container, then `depend()` should be probably named `add()` or `put()`. 

If it's not a container, but something else, then the name of this method can survive, but the name of the interface needs changing.